app/models/dinner.rb

```diff
@@ -0,0 +1,4 @@
+class Dinner < ActiveRecord::Base
+  validates :email, email: true, case_sensitive: false
+  validates :email, :dinner_name, :date, :time, :location, presence: true
```

Comment:
Usually better to organize validations by field vs. by type.  It's easier to follow and change and there can be funny interactions between validations when done this way.